mindmeld/auto_annotator.py

```diff
@@ -835,5 +901,320 @@ def valid_entity_check(self, entity):
         return Entity.is_system_entity(entity) or entity in self.supported_entity_types
 
 
-register_annotator("SpacyAnnotator", SpacyAnnotator)
-register_annotator("BootstrapAnnotator", BootstrapAnnotator)
+class NoTranslationDucklingAnnotator(Annotator):
+    """Custom Annotator class used to generate annotations."""
```

Comment:
More detailed docs here on what NoTranslationDucklingAnnotator does would be helpful. Comparing and contrasting this with the other annotators would also be great